pkg/iofog/install/local_container.go

```diff
@@ -106,6 +105,15 @@ func NewLocalAgentConfig(name string, image string, ctrlConfig *LocalContainerCo
 	if image == "" {
 		image = "docker.io/iofog/agent:" + util.GetAgentTag()
 	}
+
+	var bindings []string
+
+	if runtime.GOOS == "windows" {
```

Comment:
should this be in release before 1.4.0?

Reply:
As dicussed, comes form a cherry-pick. It is working code, I would leave it there but not market iofogctl as supporting windows yet.